### PR TITLE
Allow administrators to change users' passwords

### LINKED ISSUE #1409: Allow administrators to change users' passwords
To enable password resets, administrators need to be able to change users' passwords.

### This task is done when...
 - [ ] the "edit account" admin page allows admins to change users' passwords

### PR DESCRIPTION
**Pull request by mgwalker**
Administrators need to be able to change other users' passwords to support the password reset workflow.

### This pull request changes...
- API `PUT /users/:id` endpoint accepts passwords
- app `/admin/edit-account` has a password field
- closes #1409

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] The change has been documented
  - [x] Associated OpenAPI documentation has been updated

### This feature is done when...
- [x] Product has approved the experience

### CODE REVIEW

**Comment by mgwalker** [comment]
Implements the same UX pattern as #1412

**Comment by mgwalker** [comment]
Fundamentally I'm not opposed to that.  However, the error/success message at the top is also on the other admin forms.  I'm not sure the best way to proceed, but I worry (maybe needlessly?) about...
- changing other forms in this PR
- merging this PR and creating an inconsistent behavior

Would it make sense to create an issue to move the error/success alert in all the admin forms, so design has a chance to weigh in?  There are four of them:
- admin/create account
- admin/edit account
- manage own account
- login

**Comment by jeromeleecms** [comment]
That works for me. I'll approve this PR and spin up the new issue.

**Comment by jeromeleecms** [comment]
Spun up #1438 to address the Design review of the toast messages.

**Comment by jameshupp** [comment]
For unrelated reasons (I forgot my own user account password), I also was testing this PR. As the admin, I changed the password on my user account and the app told me it changed successfully. A few seconds later, as well as several minutes later, I tried to log in using the new password. The login attempt failed, and the app reported the username or password was not correct.

File: web/src/containers/admin/EditAccount.js [hunk 1]
```diff
@@ -109,6 +117,43 @@ class EditAccount extends Component {
...
+          <div className="ds-l-row ds-u-padding-x--2">
+            <TextField
+              label="Password"
+              ariaLabel="Current password"
+              value="********"
+              disabled
+              size="medium"
+            />
+            <div className="ds-u-clearfix">
+              <div className="ds-c-label">&nbsp;</div>
```
Comment (hbillings) [inline web/src/containers/admin/EditAccount.js]: What's up with this empty label?

Comment (mgwalker) [reply web/src/containers/admin/EditAccount.js]: It's to get the button vertically aligned with the textfield.  It seemed like the easiest way to me.  😬

**Pull request merged**